src/components/graphs/bargraph/HorizontalBar.js

```diff
@@ -71,6 +71,7 @@ const Horizontalbar = () => {
 
   return (
     <div>
+      {console.log('LOOK HERE', dataList)}
```

Comment:
Is this `console.log` strictly necessary to keep in the code now that we have the feature implemented?

Reply:
Yep that's removed with the latest commit now